cmd/autoconfigbrancher/main.go

```diff
@@ -137,6 +137,22 @@ func main() {
 		command   string
 		arguments []string
 	}{
+		{
+			command: "/usr/bin/registry-replacer",
+			arguments: []string{
+				"--github-token-path", "/etc/github/oauth",
+				"--github-endpoint", "http://ghproxy",
+				"--config-dir", "./ci-operator/config",
+				"--create-pr",
```

Comment:
This needs to be `false` :P

Reply:
Nice catch! Yea since we moved the job in autoconfigbrancher it doesn't need to create a pr. Thanks.